staging/src/k8s.io/apiextensions-apiserver/test/integration/subresources_test.go

```diff
@@ -489,24 +454,18 @@ func TestValidateOnlyStatus(t *testing.T) {
 		},
 	}
 
-	noxuDefinitions := NewNoxuSubresourcesCRDs(apiextensionsv1beta1.NamespaceScoped)
-	for i, noxuDefinition := range noxuDefinitions {
-		if i == 0 {
-			noxuDefinition.Spec.Validation = &apiextensionsv1beta1.CustomResourceValidation{
-				OpenAPIV3Schema: schema,
-			}
-		} else {
-			noxuDefinition.Spec.Versions[0].Schema = &apiextensionsv1beta1.CustomResourceValidation{
-				OpenAPIV3Schema: schema,
-			}
-			schemaWithDescription := schema.DeepCopy()
-			schemaWithDescription.Description = "test"
-			noxuDefinition.Spec.Versions[1].Schema = &apiextensionsv1beta1.CustomResourceValidation{
-				OpenAPIV3Schema: schemaWithDescription,
-			}
+	noxuDefinitions := NewNoxuSubresourcesCRDs(apiextensionsv1.NamespaceScoped)
+	for _, noxuDefinition := range noxuDefinitions {
+		noxuDefinition.Spec.Versions[0].Schema = &apiextensionsv1.CustomResourceValidation{
+			OpenAPIV3Schema: schema,
+		}
+		schemaWithDescription := schema.DeepCopy()
+		schemaWithDescription.Description = "test"
```

Comment:
I have no idea why we had set the decription at all. Would completely remove that `schemaWithDescription` code here and see whether it's green.